src/modules/m_quit.c

```diff
@@ -91,7 +91,24 @@ DLLFUNC int  m_quit(aClient *cptr, aClient *sptr, int parc, char *parv[])
 		char *s = comment;
 		Hook *tmphook;
 		if (STATIC_QUIT)
-			return exit_client(cptr, sptr, sptr, STATIC_QUIT);
+		{
+			/* Lets not let unregistered users spam us even without STATIC_QUIT
+			 * except opers. -katsklaw
+			 * Removed anti-spam-quit-message-time and integrate it with STATIC_
+			 * QUIT and STATIC_PART. Remodified katsklaw's code
+			 * TODO: update config file and docs -dboyz
+			 */
+			if (STATIC_QUIT_PART_TIME == -1)
+			{
+				if!(IsLoggedIn(sptr) && STATIC_QUIT_PART_SET)
+					return exit_client(cptr, sptr, sptr, STATIC_QUIT);
+			}
+			else
```

Comment:
Please use brackets with this else (so else { .... } ) since its next statement spans multiple lines